modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -221,7 +221,7 @@ def perform_pdf_upload(location, file_path, metadata, form)
 
       def upload_pdf_to_s3(id, file_path, metadata, submission, form)
         config = SimpleFormsApi::FormRemediation::Configuration::VffConfig.new
-        attachments = get_form_id == 'vba_20_10207' ? form.attachment_guids : []
```

Comment:
does this need to be removed, is it used any more?

Reply:
This was actually the cause of the bug. This piece of logic was a holdover from an earlier version of the submission backup logic. The bug was caused by a collection of attachment IDs being processed as attachment file paths. This collection should have always been a collection of file paths.